src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
```

Comment:
Directly importing `usersFromServer` here goes against the guideline of preparing data in one place, typically the `App` component. Consider passing the user data as props to this component instead.